modules/databend/src/test/java/org/testcontainers/DatabendTestImages.java

```diff
@@ -0,0 +1,7 @@
+package org.testcontainers;
+
+import org.testcontainers.utility.DockerImageName;
+
+public interface DatabendTestImages {
+    DockerImageName DATABEND_IMAGE = DockerImageName.parse("datafuselabs/databend:latest");
```

Comment:
Let's avoid using latest and instead use a fixed version.

Reply:
Thanks! fixed.